eggplant_project/settings/base.py

```diff
@@ -15,6 +15,10 @@
 from django.utils.translation import ugettext_lazy as _
 from django.contrib.messages import constants as messages_constants
 
+COOP_NAME = 'KBHFF'
```

Comment:
If we are talking about software design patterns than to my understanding of it the problem you are trying to describe that we may potentially have does not have attributes of it. Convention, on the other hand, is a way in which something is usually done or a suggested way of doing things, as opposed to required way of doing things.